Applications/DataExplorer/DataView/SHPImportDialog.cpp

```diff
@@ -24,11 +24,20 @@
 #include <QRadioButton>
 #include <QVBoxLayout>
 
-SHPImportDialog::SHPImportDialog(std::string filename, GEOModels* geoModels, QDialog* parent)
-	: QDialog(parent), _buttonBox(NULL), _layout(NULL), _shpContentLabel(NULL), _nameLabel(NULL),
-	  _listName(new QLineEdit()), _choice1(NULL), _choice2(NULL), _filename(filename),
-	  _fileType(0),
-	  _shpInterface(new FileIO::SHPInterface(geoModels))
+SHPImportDialog::SHPImportDialog(std::string filename,
+                                 GeoLib::GEOObjects* geo_objects,
+                                 QDialog* parent)
+    : QDialog(parent),
+      _buttonBox(NULL),
```

Comment:
It is possible to substitute `NULL` with `nullptr` in the complete initializer list.